### PR TITLE
Feat: Vibes Console [Applications Center]

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- vibes console applications center flows

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/vibes-console/src/components/app-routes/index.tsx [hunk 1]
```diff
@@ -1,29 +1,88 @@
...
+  MY_APPLICATIONS,
+  MY_APPLICATION_DETAIL,
+  WITHDRAW_APPLICATION,
+} from '../../routes';
 
 const rootRoute = rootRouteWithContext()({
   component: Outlet,
 });
 
 const applicationsRoute = new Route({
```
Comment (quininez) [inline ui/apps/vibes-console/src/components/app-routes/index.tsx]: Route is deprecated, should be replaced with createRoute, also rootRouteWithContext is deprecated [see reference](https://tanstack.com/router/latest/docs/framework/react/api/router)

File: ui/apps/vibes-console/src/utils/dummy-data.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import { randomDateBetweenValues } from '@akashaorg/design-system-core/lib/utils';
```
Comment (quininez) [inline ui/apps/vibes-console/src/utils/dummy-data.ts]: remove dummy data from apps, keep it for stories

File: ui/design-system-core/src/utils/time.ts [hunk 3]
```diff
@@ -47,4 +47,11 @@ const formatRelativeTime = (date: string, locale?: string) => {
   return '';
 };
 
-export { formatDate, formatDateShort, formatRelativeTime };
+const randomDateBetweenValues = (start = 'Jan 01 2020', end = 'Dec 31 2020') => {
```
Comment (quininez) [inline ui/design-system-core/src/utils/time.ts]: if this is used for dummy data, remove it from here

**Review by quininez — CHANGES_REQUESTED** [review]

**Comment by quininez** [comment]
can you also add react-helmet to this app? you can add it in the app.tsx, check for example extensions app

File: ui/design-system-core/src/utils/time.ts (hunk 3 above)
Comment (josenriagu) [reply ui/design-system-core/src/utils/time.ts]: You suggested earlier that I move that method to time utils in DS core. could be used any other place in future

Comment (quininez) [reply ui/design-system-core/src/utils/time.ts]: it seems very specific for the vibes app, even if I suggested this, which I don't recall, I consider that it's better to be in some util related to dummy data, just like the other generator functions

Comment (josenriagu) [reply ui/design-system-core/src/utils/time.ts]: okay, I will refactor it

**Review by quininez — APPROVED** [review]

**Pull request merged**